### ISSUE TITLE
Asking for loss value

### ISSUE THREAD

**Issue by coolmakersss** [rhett-chen/graspness_implementation#31]
Hi, @rhett-chen,

Thank you very much for your excellent code.

The AP value of my own training results is only about half of what is reported in the paper and I'm debugging. 

I wanted to ask if you remember the approximate values of each loss function after they converged, not too precisely. If you could provide them, I would be very grateful.

Thank you!

Best regards

**Comment by rhett-chen** [comment]
----epoch: 010  ---- batch: 5400 ----
mean graspable_count_stage1: 2026.875000
mean loss/overall_loss: 0.768805
mean loss/stage1_graspness_loss: 0.001724
mean loss/stage1_objectness_loss: 0.068601
mean loss/stage2_view_loss: 0.005997
mean loss/stage3_score_loss: 0.003021
mean loss/stage3_width_loss: 0.003791

**Comment by coolmakersss** [comment]
Thank you very much!
